charters/bcdc-cloud-computing-environment.md

```diff
@@ -0,0 +1,41 @@
+# BCDC Cloud Computing Environment (Terra)
+## Description
+We connect, develop, add value to, and train our community around the BCDC Cloud Computing Environment. This is a cloud-native environment that enables many functionality including cloud storage, scalable data processing pipelines, and horizontally scalable data-science environments.
+
+# Roles
+[Principal Investigator] Timothy Tickle (ttickle@broadinstitute.org)   
+[Pipelines Development Product Manager] Kylee Degatano (email@email.com)   
+[Customer Success and Education] Salin Thomas (salin@broadinstitute.org)  
+[Project Manager] Cara Mason (cara@broadinstiute.org)
+
+## Definitions
+**pipeline:** A collection of one or more functional tasks that operate on input data and, from that, transform the input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
+
+## Objectives
+- To create and maintain a cloud native, horizontally scalable environment for use by the BICCN Community.   
+- To work with community member to bring data processing pipelines used by the community or of value to the community into the cloud computing envrinment so they may be operated at scale.   
+
+## In-scope
```

Comment:
Should something be considered in scope to maintain and upgrade pipelines depending on updates to genomes etc (not sure if this happens regularly enough to impact us for this grant)

Reply:
Yes, absolutely

Reply:
Added.

Reply:
Will add.